graph/pull_v2_test.go

```diff
@@ -167,28 +167,4 @@ func TestValidateManifest(t *testing.T) {
 	if err == nil || !strings.HasPrefix(err.Error(), "image verification failed for digest") {
 		t.Fatal("expected validateManifest to fail with digest error")
 	}
-
-	// Manifest with no signature
-
-	expectedWholeFileDigest := "7ec3615a120efcdfc270e9c7ea4183330775a3e52a09e2efb194b9a7c18e5ff7"
```

Comment:
Yes. The Unmarshal call a few lines below fails because the current `SignedManifest` unmarshal method uses the signature section as part of the unmarshaling process. If the signature is missing, JSON unmarshaling fails.